src/DeviceInterface/InterfacePrototype.js

```diff
@@ -225,10 +232,11 @@ class InterfacePrototype {
     }
 
     async startInit () {
+        await this.refreshSettings()
```

Comment:
We're now initialising the settings before doing anything else, so the listeners can access them.